docs/sdk/src/reference_docs/omni_node.rs

```diff
@@ -177,9 +177,35 @@
 //! [This](https://github.com/paritytech/polkadot-sdk/issues/5565) future improvement to OmniNode
 //! aims to make such checks automatic.
 //!
+//! ### Runtime conventions
+//!
+//! Given OmniNode purpose is to be compatible with most Polkadot parachains, we thought it is
+//! also achievable to guide how the developed parachain's runtimes should be composed. The
+//! runtimes should contain a set the basis for a functional parachain in the context of any
+//! node. We check for these pallets at OmniNode start-up, parsing through the runtime metadata.
+//! The checks are best effort, given the runtime code is custom, and can be considered as lint
+//! rules which will generate warning level logs in the Omni Node log file, when not complied with.
+//!
+//! The checks were introduced mainly to detect incomplete runtime implementations, from a
+//! parachain compatibility standpoint, but they are also useful to ensure deeper compatibility
+//! with Omni Node, which should be compatible with the majority of the parachains in the
+//! ecosystem, and enable/disable certain features given a chain spec, runtime metadata, or a
+//! set of CLI flags. Such built-in checks should improve the developer experience by providing
+//! useful feedback, developers setup their runtime correctly for a parachain use case.
+//!
```

Comment:
IMO this paragraph can be removed

Reply:
Indeed, done: [0ba6ecb](https://github.com/paritytech/polkadot-sdk/pull/6450/commits/0ba6ecb244c1ae43bfc6262e2c1143bf3f735410)